main.go

```diff
@@ -160,6 +156,13 @@ func handleRequests() {
 		classrooms.DeleteClassroom(w, r, client.Database("schedule_db").Collection("classrooms"))
 	}).Methods(http.MethodDelete)
 
+}
+
+func handleRequests() {
```

Comment:
Will need to pull in and resolve any changes and conflicts from dev. Some of the files here have changed pretty substantially. Otherwise good work

Reply:
Thanks, sounds good. I will pull from dev.